test/test_common/environment.h

```diff
@@ -93,6 +93,18 @@ class TestEnvironment {
    */
   static const std::string& nullDevicePath();
 
+  /**
+   * Obtain platform specific new line character(s)
+   * @return const std::string& platform specific new line character(s)
```

Comment:
nit: s/std::string&/absl::string_view

Reply:
thanks good catch, thats what happens when you push quickly :)

Reply:
fixed in da2fc69009d6df2da4ffa8b2e3b300154df0e26a